packages/website/ts/pages/explore.tsx

```diff
@@ -0,0 +1,390 @@
+
+export interface ExploreProps {}
+
+interface ExploreModifierOptions {
+    filter?: ExploreFilterMetadata;
+    query?: string;
+    isEditorialShown?: boolean;
+    tilesOrdering?: ExploreTilesOrdering;
+}
+
+export class Explore extends React.Component<ExploreProps> {
+    public state = {
+        isTilesLoading: false,
+        tiles: [] as ExploreTile[],
+        tilesOrdering: ExploreTilesOrdering.Popular,
+        isEditorialShown: true,
+        filters: FILTERS,
+        query: '',
+    };
+
+    private readonly _debouncedChangeSearchResults: (query: string) => void;
+
+    constructor(props: ExploreProps) {
+        super(props);
+        this._debouncedChangeSearchResults = _.debounce(this._changeSearchResults, 300);
+    }
+
+    // tslint:disable-next-line:async-suffix
+    public async componentDidMount(): Promise<void> {
+        await this._loadEntriesAsync();
+        await this._setFilter('all');
+    }
+
+    public render(): React.ReactNode {
+        return (
+            <SiteWrap theme="light">
+                <DocumentTitle {...documentConstants.EXPLORE} />
+                <ExploreHero query={this.state.query} onSearch={this._setNewQuery} />
+                <Section isPadded={false} padding={'0 0 60px 0'} maxWidth={'1150px'}>
+                    <ExploreToolBar
+                        onFilterClick={this._setFilter}
+                        filters={this.state.filters}
+                        // editorial={this.state.isEditorialShown}
+                        // onEditorial={this._onEditorial}
+                        orderings={ORDERINGS}
+                        activeOrdering={this.state.tilesOrdering}
+                        onOrdering={this._onOrdering}
+                    />
+                    <ExploreGrid tiles={this._generateTilesFromState()} />
+                </Section>
+            </SiteWrap>
+        );
+    }
+
+    // tslint:disable-next-line:no-unused-variable
+    private readonly _onEditorial = async (newValue: boolean): Promise<void> => {
+        const newTiles = await this._generateTilesWithModifier(this.state.tiles, ExploreTilesModifiers.Editorial, {
+            isEditorialShown: newValue,
+        });
+        this.setState({ isEditorialShown: newValue, tiles: newTiles });
+    };
+
+    private readonly _onOrdering = async (newValue: string): Promise<void> => {
+        this.setState({ tilesOrdering: newValue });
+        const newTiles = await this._generateTilesWithModifier(this.state.tiles, ExploreTilesModifiers.Ordering, {
+            tilesOrdering: newValue as ExploreTilesOrdering,
+        });
+        this.setState({ tilesOrdering: newValue, tiles: newTiles });
+    };
+
+    private readonly _launchInstantAsync = (params: ExploreProjectInstantMetadata): void => {
+        zeroExInstant.render(params, 'body');
+    };
+
+    private readonly _onAnalytics = (project: ExploreProject, action: ExploreAnalyticAction): void => {
+        switch (action) {
```

Comment:
If possible, it would be cool to track whether people are filtering and searching as well.